--- plasma-phone-components-0.7+git20200221.orig/containments/taskpanel/package/contents/ui/TaskSwitcher.qml
+++ plasma-phone-components-0.7+git20200221/containments/taskpanel/package/contents/ui/TaskSwitcher.qml
@@ -183,20 +183,4 @@ NanoShell.FullScreenOverlay {
             }
         }
     }
-
-    PlasmaComponents.RoundButton {
-        id: showDesktopButton
-        height: units.iconSizes.large
-        width: height
-        anchors {
-            horizontalCenter: parent.horizontalCenter
-            bottom: parent.bottom
-        }
-        icon.name: "start-here-kde"
-        onClicked: {
-            currentTaskIndex = -1;
-            window.hide();
-            plasmoid.nativeInterface.showDesktop = true;
-        }
-    }
 }
